manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml

```diff
@@ -54,7 +55,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: csi-snapshotter
-          image: k8s.gcr.io/sig-storage/csi-snapshotter:v6.0.1
+          image: k8s.gcr.io/sig-storage/csi-snapshotter:v6.1.0
```

Comment:
:nit: we should remove the stale reference of v1beta1 here https://github.com/kubernetes/cloud-provider-openstack/blob/master/charts/cinder-csi-plugin/values.yaml#L130 , I guess everything else is moved to v1